packages/OreIdReact/src/components/OreidProvider/OreIdProvider.tsx

```diff
@@ -1,24 +1,26 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { OreId, UserData } from "oreid-js";
 import { OreIdContext } from "src/OreIdContext";
 import { useState } from "react";
 import { Observables } from "../Observables";
 
 interface Props {
 	oreId: OreId;
-
 	children: React.ReactNode;
 }
 export const OreidProvider: React.FC<Props> = ({ children, oreId }) => {
 	const [user, setUser] = useState<UserData | undefined>(undefined);
 	const [isLoggedIn, setIsLoggedIn] = useState(false);
 	const [accessToken, setAccessToken] = useState<string | undefined>(undefined);
 
-	if (!oreId.popup) {
-		throw new Error(
-			"oreId popup must be initialized. Make sure you added the plugin and called oreId.init()"
-		);
-	}
+	useEffect(() => {
+		if (typeof window === "undefined") {
+			return;
+		}
+		if (!oreId.isInitialized) {
```

Comment:
My only concern here is backwards compatibility. But I see that minor version was increased so it should be ok. Another option would be to opt in like: `<OreIdProvider init={true}>`

Reply:
This change shouldn't break anything.
The worst case scenario is calling "init" twice, which will do nothing